pylabrobot/plate_reading/plate_reader.py

```diff
@@ -94,11 +98,15 @@ async def read_luminescence(
   @need_setup_finished
   async def read_absorbance(
     self, wavelength: int, wells: Optional[List[Well]] = None, **backend_kwargs
-  ) -> List[List[Optional[float]]]:
-    """Read the absorbance from the plate in OD, unless otherwise specified by the backend.
+  ) -> List[Dict[Tuple[int, int], Dict]]:
+    """Read the absorbance from the plate reader.
 
     Args:
       wavelength: The wavelength to read the absorbance at, in nanometers.
+
+    Returns:
+      A list of dictionaries, one for each timepoint. Each dictionary has a key (wavelength, 0)
+      and a value containing the data, temperature, and time.
```

Comment:
why not use the wavelength as the key directly?

Reply:
Indexing with `,0` is strange in the api. They are slightly different reads, I don't see why we would use the same format for all